featuretools/utils/gen_utils.py

```diff
@@ -117,3 +120,52 @@ def check_schema_version(cls, cls_type):
         # Check if saved has older major version.
         if current[0] > saved[0]:
             raise RuntimeError(error_text_outdated)
+
+
+def use_smartopen_es(file_path, path, transport_params=None, read=True):
+    if read:
+        with open(path, "rb", transport_params=transport_params) as fin:
+            with open(file_path, 'wb') as fout:
+                for line in fin:
+                    fout.write(line)
+    else:
+        with open(file_path + ".tar", 'rb') as fin:
```

Comment:
I think instead of modifying the file path by adding a tar extension we should update the docstrings to make it clear to users that uploading to s3 creates a tar archive instead of a directory.